python/google/protobuf/text_format.py

```diff
@@ -890,7 +890,7 @@ def _MergeField(self, tokenizer, message):
       expanded_any_sub_message = _BuildMessageFromTypeName(packed_type_name,
                                                            self.descriptor_pool)
       # Direct comparison with None is used instead of implicit bool conversion
-      # to avoid false positives with falsy initial values, e.g. for
+      # to avoid false positives with "falsey" initial values, e.g. for
```

Comment:
Can we put the typo fixes and other cleanups into a separate PR? I'd like to keep the clarity of "just fix the UTF8 problem".

Reply:
Definitely. I was really just trying to get a green build. But looks like that won't happen anyway, so I'll pull this commit out.

Reply:
I've dropped this commit.